CONTRIBUTING.md

```diff
@@ -59,6 +59,15 @@ To contribute to this repo, you'll use the _Fork and Pull_ model common in many
 - For details on the local process, check out the [GitHub flow](https://docs.github.com/en/get-started/using-github/github-flow) documentation from GitHub and [The GitHub Workflow Guide](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) documentation from Kubernetes.
 - For details on contributing using the GitHub webpage UI, see [Contributing using the GH UI](docs/contributing_via_GH_UI.md).
 
+This repository uses [submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to incorporate the [taxonomy schema](https://github.com/instructlab/schema.git).
+So, when using a local clone of this Git repository locally, be sure to use the [`--recurse-submodules`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---recurse-submodulesltpathspecgt) option on the `git clone` command and the `git pull` command when pulling updates from the remote repository.
```

Comment:
```suggestion
So, when using a local clone of this Git repository, be sure to use the [`--recurse-submodules`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---recurse-submodulesltpathspecgt) option on the `git clone` command and the `git pull` command when pulling updates from the remote repository.
```

Reply:
It already says "when using a local clone". So a second local seems redundant.

Reply:
I think you read too fast @bjhargrave I suggested *removing* locally because it is indeed redundant.

Reply:
There is a reason the standard policy is to let the commenter mark comments resolved. :-)